polymetis/tests/scripts/0_joint_positions.py

```diff
@@ -35,35 +35,38 @@ def test_new_joint_pos(robot, joint_pos_desired):
     print("=== RobotInterface.move_to_joint_positions ===")
     delta_joint_pos_desired = torch.Tensor([0.0, 0.0, 0.0, 0.5, 0.0, -0.5, 0.0])
     joint_pos_desired = joint_pos + delta_joint_pos_desired
+    time_to_go = 3.0
```

Comment:
Not sure I understand the reasoning behind this file's changes.

Reply:
Before the default time_to_go in RobotInterface was set to 3 seconds because previously calling move_to_ee_pose with small movements might still result in large joint motions. Since we are actively using IK to figure out the joint motion now I shortened it to 1 second so we wouldn't need to wait 3 seconds for any tiny movement.
This test script originally does not explicitly assign time_to_go to the joint commands (thus using the default time_to_go), so I added back 3.0 seconds of time_to_go to retain the same behavior as before.